src/index.js

```diff
@@ -7,8 +10,57 @@ const getTop = function(element, start) {
 
 const smoothScrollCtx = Symbol('smoothScrollCtx')
 
+export const smoothScroll = function ({ scrollTo, offset = 0, duration = 500, container = window, updateHistory = true, hash }) {
```

Comment:
I do not see a need to export this anymore. In which case the duplication of default param values (with `defaultValues` obj) can also be removed:

```js
function _smoothScroll({ scrollTo, offset, duration, container, updateHistory, hash })
```
may be enough since the method will only be called internally in the binding for `$smoothScroll` below.

Reply:
This makes sense - i've just pushed that change. Thanks for your help with this @a-kriya, I'm pretty new to open source contribution so thank you for making it such a good experience.

Reply:
I'm glad to help. This was something that I needed too, and bringing it to a place where it can be merged and published is great, so you could say that it was out of self-interest, haha. Thanks for working on this. Now, let's see if @Yuliang-Lee has any other concerns here.